docs/dev-machine-setup.md

```diff
@@ -21,7 +21,27 @@ outlined in the [Getting Started](getting-started.md) instructions).
 
 ### AWS
 
-`[TODO]`
+Firecracker development environment on AWS can be setup using bare metal instances. Follow these steps to create a bare metal instance.
+
+1. If you don't already have an AWS account, create one: https://portal.aws.amazon.com/billing/signup.
```

Comment:
nit: do not use bare URL in markdown. One suggestion would be to re-write it as:
```
... create on using the [AWS portal](https://portal.aws.amazon.com/billing/signup).
```
or whatever else sounds better.